bom/runtime/pom.xml

```diff
@@ -169,6 +169,7 @@
         <spring-data.version>2.1.9.RELEASE</spring-data.version>
         <spring-security.version>5.2.0.RELEASE</spring-security.version>
         <spring-boot.version>2.1.10.RELEASE</spring-boot.version>
+        <quarkus-spring-api.version>0.0.1-SNAPSHOT</quarkus-spring-api.version>
```

Comment:
I don't really like the `0.0.1` TBH. I would probably go for `0.1` as the first release

Reply:
Why not use the version of Spring they are based on? That would be far better IMHO.

Reply:
That's an option as well. We did discuss doing something like `2.1.3.Alpha1`, `2.1.3.CR1` etc, but we didn't settle on what the proper strategy should be

Reply:
Ah you would base it on Spring Boot. My inclination would have been to really use the Spring version of the related project. That way, you know what you are using.

Reply:
But thinking a bit more about it, it has been so long I used Spring that nowadays people might always speak in terms of Spring Boot versions.

I just think that it could be cool to check the version of Spring Security APIs and be able to easily find the related javadoc.

Reply:
I'm not saying we need the Spring Boot version, I was just giving an example. I personally think that using the actual Spring component version instead of the Spring Boot version makes more sense as you say.
But then again there is the argument that for Quarkus, the Spring version doesn't really matter, since we don't implement evething anyway, nor are we affected by bugs in Spring projects

Reply:
I don't think that's a good call. They might add some features in new versions and knowing that we don't support them is good IMHO.

We could maybe track the major.minor version only.

Reply:
That's probably a good compromise

Reply:
please, see https://github.com/quarkusio/quarkus/pull/7169#discussion_r378712096

Reply:
Ok, so 
`<quarkus-spring-api.version>5.1-SNAPSHOT</quarkus-spring-api.version>`?
as :
quarkus-spring-*-api --> `Spring Boot 2.1.12` --> `Spring Framework 5.1.13.RELEASE`

Reply:
@aureamunoz I'm not sure what you mean `quarkus-spring-*-api --> Spring Boot 2.1.12 --> Spring Framework 5.1.13.RELEASE`.

Let's do `<quarkus-spring-web-api.version>5.1.Alpha1</quarkus-spring-web-api.version>` for Spring Web 5.1.x.RELEASE. Does that make sense?

Reply:
Yes, that makes sense for me.

I meant that currently the quarkus spring dependencies (`quarkus-spring-web-api `, `quarkus-spring-di-api` and so on) depend on Spring Boot 2.1.12.RELEASE which depend on Spring 5.1.13.RELEASE. 

Reply:
Got it, thanks.

I don't think we need to care about the Spring Boot version. The more I think about it the more I agree with @gsmet's proposal of using only the major and minor version of the actual Spring artifact

Reply:
Yes, I definitely agree with you. The commit is in progress ;-)

Reply:
We need to consider that we're likely to make multiple releases that will use the same Spring Boot version.
It's because we only picked a subset of classes from Spring that we need at the moment. If we'll add a new feature to an extension, we might need to include new classes to the dependencies jar.

Reply:
Yes of course. My proposal is that we just add the proper qualifiers: `Alpha1`, `Alpha2`, `Beta1`, `Beta2`, `Final`, `CR1`, `CR2` etc.

Reply:
I don't think that will work. Let's say you have a Final and you end up needing a new annotation to implement a new feature in Quarkus?

Reply:
That's what CR1 is supposed to be for.

Reply:
I assume we'll have to do similar dependencies bundle for Spring Boot as well (e.g. for spring boot properties extension). Are we going to use spring boot version for that?